README.md

```diff
@@ -77,6 +77,9 @@ There are currently 3 options to run Metorikku.
 * Download the [last released Standalone JAR](https://github.com/YotpoLtd/metorikku/releases/latest)
 * Run the following command:
 `java -Dspark.master=local[*] -cp metorikku-standalone.jar com.yotpo.metorikku.Metorikku -c config.yaml`
+* Also job in a JSON format is supported, run following command:
+`java -Dspark.master=local[*] -cp metorikku-standalone.jar com.yotpo.metorikku.Metorikku --job "{*}"`
+ 
```

Comment:
Also add a comment in the streaming section that we support streaming from file sources (explain what it is, you can copy from spark's docs)